php/blocks/block-coauthor-name/class-block-coauthor-name.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * Co-Author Name Block
+ * 
+ * @package Automattic\CoAuthorsPlus
+ * @since 3.6.0
+ */
+
+namespace CoAuthors\Blocks; 
+
+use WP_Block;
+
+/**
+ * Block CoAuthor Name
+ * 
+ * @package CoAuthors
+ */
+class Block_CoAuthor_Name {
+	/**
+	 * Register Block
+	 *
+	 * @since 3.6.0
+	 */
+	public static function register_block(): void {
+		register_block_type(
+			dirname( COAUTHORS_PLUS_FILE ) . '/build/blocks/block-coauthor-name',
+			array(
+				'render_callback' => array( __CLASS__, 'render_block' ),
+			)
+		);
+	}
+
+	/**
+	 * Render Block
+	 *
+	 * @since 3.6.0
+	 * @param array    $attributes
+	 * @param string   $content
+	 * @param WP_Block $block
+	 * @return string
+	 */
+	public static function render_block( array $attributes, string $content, WP_Block $block ): string {
+
+		$author = $block->context['co-authors-plus/author'] ?? array();
+
+		if ( empty( $author ) ) {
+			return '';
+		}
+
+		$display_name = $author['display_name'] ?? '';
```

Comment:
Minor, can we add a bit of escaping here for safety?

```suggestion
		$display_name = esc_html( $author['display_name'] ) ?? '';
```

I'm not sure if it's possible to get HTML into a display name, but since it's being directly rendered to the page this makes sanitization explicit.

Reply:
Maybe this should be all handled in the [`render_element()` method](https://github.com/Automattic/Co-Authors-Plus/pull/997/files/5638d641eb7ae8ed469a25c3c857f7a427e40fcf?show-viewed-files=true&file-filters%5B%5D=#diff-1d30aedc4b32a3eeffc41b7b7ff74154b84143c9d5793fc5e5a2baaef396f9a9R23-R25)?

Reply:
Addressed in https://github.com/Automattic/Co-Authors-Plus/pull/997/commits/73f7c614224cc49bc2ec778bff90c14851f18a77 along with a docblock comment for `render_element` clarifying you can pass it HTML.